source/classes.rst

```diff
@@ -4,6 +4,11 @@ Classes
 Python 3 drops support for “old-style” classes, and introduces dedicated syntax
 for metaclasses. Read on for details.
 
+
+.. index:: object; as base class
```

Comment:
I think it's relevant here – `object` can mean many things, even in the context of porting. I wanted to be clear that this is about `object` added as a base class as part of porting,